web/source/kmwembedded.ts

```diff
@@ -121,6 +121,9 @@ namespace com.keyman.osk {
     let util = keyman.util;
     let device = util.device;
 
+    //TODO: Refactor together with kmwnative.ts:adjustHeights function
+    //TODO: Why do we factor in the banner here but not in kmwnative.ts:adjustHeights?
```

Comment:
Hmm.  That's a good question; I can't recall at this point.  Let's make it an issue for the beta cycle, since there's no reason for it to hold this PR back.